config/manager/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: us.gcr.io/k8s-artifacts-prod/capi-openstack/capi-openstack-controller:v0.2.0
+      - image: gcr.io/k8s-staging-capi-openstack/capi-openstack-controller-amd64:dev
```

Comment:
you might want to modify this later or stick to :dev?

Reply:
I would stick to dev wenn the release YAML is generated it is set to the production image:
```
# Makefile release target
$(MAKE) set-manifest-image MANIFEST_IMG=$(PROD_REGISTRY)/$(IMAGE_NAME) MANIFEST_TAG=$(RELEASE_TAG)
```